ui_curses.c

```diff
@@ -302,6 +303,7 @@ enum {
 };
 
 static struct format_option track_fopts[NR_TFS + 1] = {
+	DEF_FO_TIME('Z', "albumduration", 0),
```

Comment:
Good point - to be honest I wasn't entirely clear on how format options worked when I added that char code. I've removed it and adjusted the default format string in `options.c`. I also added the key to the `Doc/cmus.txt` man page.